.eslintrc.js

```diff
@@ -75,6 +75,10 @@ module.exports = {
         tsx: 'never',
       },
     ],
+    'filenames/match-regex': [
+      2,
+      '^[.a-z0-9][a-z0-9-]+(?:\\.[^.]+)?(?:\\.stub\\..*|\\.spec\\..*)?$',
```

Comment:
Sorry, I'm not sure I understand the regex. What are we checking for stub and spec? Wouldn't be `'^[.a-z0-9-]+$'` enough to check the kebab case?

Reply:
Yep, you are right, that's enough.